### PR TITLE
add translation indonesia readme

### PR DESCRIPTION
**Pull request by artmxra7**
* 📝 Translates README to Indonesia

* 🔧 Updates contributions

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,5 +11,6 @@ packaged
 coverage
 public/service-worker.js
 private-key.json
-
+.yarn-error.log
```
Comment (mfix22) [inline .gitignore:14]: Would you mind updating the .gitignore to just ignore all `.log` files?

```
*.log
```

File: README.md [hunk 2]
```diff
@@ -71,6 +71,11 @@
       <td>
         <a href="/docs/README.tw.md">臺灣</a>
       </td>
+      <td>
```
Comment (mfix22) [inline README.md:74]: Can you reorganize the table such that there are two rows, one row of `9` and one row of `8? With "Add +" as the last one?

**Review by mfix22 — COMMENTED** [review]
Thanks so much for doing this! Couple nits, and we should be good to go!

File: .gitignore (hunk 1 above)
Comment (artmxra7) [reply .gitignore:14]: wow cool, i mind only ignore `yarn-error.log` but ignore all `.log` really cool

File: README.md (hunk 2 above)
Comment (artmxra7) [reply README.md:74]: oh yes, done in my last commi :)

**Review by mfix22 — APPROVED** [review]
🖤

**Pull request merged**